docs/development/build-instructions-windows.md

```diff
@@ -113,3 +113,10 @@ This can happen during build, when Debugging Tools for Windows has been installe
 ### ImportError: No module named win32file
 
 Make sure you have installed `pywin32` with `pip install pywin32`.
+
+### build scripts occasionally hang until keypress
```

Comment:
nit: this should be :

```
### Keypress Troubleshooting

Build scripts occasionally hang until keypress. This bug is a "feature" of Windows' command prompt. It happens when clicking inside the prompt window with
`QuickEdit` enabled and is intended to allow selecting and copying output text easily.
Since each accidental click will pause the build process, you might want to disable this
feature in the command prompt properties.
```

Reply:
imo, if I searched for this issue, the original heading would feel more suitable to me

Reply:
that's why i moved the original text to the subheading: it'd still be greppable, but the header style now fits in with the style of all the rest of our documentation headers.

Reply:
how about `Building stuck until keypress`? - tbh none of them will fit the style bc all other headings are error messages.. and we don't get a message here 😄

Reply:
@codebytere so what's preferred?

Reply:
Let's go with `Build Script Hang on Keypress`?